frontend/src/features/properties/map/activity/detail/content/formContent.tsx

```diff
@@ -0,0 +1,53 @@
+import { Input } from 'components/common/form';
+import { useFormikContext } from 'formik';
+import { Col, Row } from 'react-bootstrap';
+import { withNameSpace } from 'utils/formUtils';
+import * as Yup from 'yup';
+
+import { ActivityModel } from '../models';
+import { ActivityTemplateTypes, IActivityFormContent } from './models';
+export const formContent = new Map<ActivityTemplateTypes, IActivityFormContent>([
+  [
+    ActivityTemplateTypes.SURVEY,
+    {
+      header: 'Survey',
+      initialValues: { test: '' },
+      EditForm: () => (
+        <>
+          <b>TEST SURVEY FORM</b>
+          <Row>
+            <Col>
+              <Input field={withNameSpace('activityData', 'test')}></Input>
+            </Col>
+          </Row>
+        </>
+      ),
+      ViewForm: () => {
+        const { values } = useFormikContext<ActivityModel>();
```

Comment:
if its a view, why do we need formik? Can we pass the view props instead?

Reply:
If you look at the ActivityContainer the parent of the form wraps either the View or Edit form with formik so both share the formik context. Let's discuss on our meeting about pattern for View/Edit forms like these

Reply:
you could pass the props instead, either way you will be within the context of the parent formik instance.